File: Aprendendo-Java/Exercícios/retangulo/src/retangulo/Retangulo.java

package retangulo;
import java.util.Scanner;
import java.util.Locale;

public class Retangulo {

    
    public static void main(String[] args) {
        // TODO code application logic here
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);
        
        double base, altura, diagonal, area, perimetro;
        
        System.out.println("Digite a base do retangulo: ");
        base = sc.nextDouble();
        
        System.out.println("Digite a altura do retangulo: : ");
        altura = sc.nextDouble();
        
        area = base * altura;
        
        perimetro = 2*(base + altura);
        
        diagonal = Math.sqrt(Math.pow(base,2)+ Math.pow(altura,2));
        
        
        System.out.println("Area: " + area);
        System.out.println("Perimetro: " + String.format("%.4f",perimetro));
        System.out.println("Diagonal: " + String.format("%.4f", diagonal));
        
        
    }
    
}

File: Aprendendo-Python/Exercícos/retangulo.py
import math
base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = (base * altura)

perimetro = (base + altura) * 2

diagonal = math.sqrt(base**2 + altura**2)

print(f"Area: {area:.4f}")
print(f"Perimetro: {perimetro:.4f}")
print(f"Diagonal: {diagonal:.4f}")